### PR TITLE
First commit boi

### PR DESCRIPTION
**Pull request by aprajapati26**
# Creating pull request in order to merge changes with master. Will likely need review in case something went wrong.

Tasks need:
- [x] code review
- [x] more commits
- [x] final code review
- [x] Merge

### CODE REVIEW

**Review by waqarpseg — APPROVED** [review]
good stuff boyyyy

File: _posts/0000-01-10-aprajapati26.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is the best picture every !!
```
Comment (githubteacher) [inline _posts/0000-01-10-aprajapati26.md:6]: I agree. Do you think that your sentence might look better with emojis? ❤️

**Review by githubteacher — CHANGES_REQUESTED** [review]
Looks great. Left some feedback. ❌

File: _posts/0000-01-10-aprajapati26.md (hunk 1 above)
Comment (githubteacher) [inline _posts/0000-01-10-aprajapati26.md:6]: What about this one?

```suggestion
This is the best picture ever!! ✨ 🎉 🎈 
```

**Review by githubteacher — APPROVED** [review]

**Pull request merged**